src/test/java/com/ebivariation/contigalias/controller/AdminControllerIntegrationTest.java

```diff
@@ -66,7 +65,6 @@ public void getAssemblyOrFetchByAccessionGCA() throws Exception {
         this.mockMvc.perform(get("/contig-alias-admin/assemblies/{accession}",
                                  entity.getGenbank()))
                     .andExpect(status().isOk())
```

Comment:
I'd prefer an actual check that it's not present. Would this work?
```suggestion
                    .andExpect(status().isOk())
                    .andExpect(jsonPath("$.id").doesNotExist())
```